oak/server/storage/fixed_nonce_generator.h

```diff
@@ -33,8 +35,18 @@ class FixedNonceGenerator : public asylo::NonceGenerator<kAesGcmSivNonceSize> {
   // Called by asylo::AesGcmSiv::Seal.
   asylo::Status NextNonce(const std::vector<uint8_t>& key_id, AesGcmSivNonce* nonce) override {
     CHECK(nonce != nullptr);
-    std::copy(key_id.begin(), key_id.end(), nonce->begin());
-    std::copy(datum_name_.begin(), datum_name_.end(), nonce->begin() + key_id.size());
+
+    SHA256_CTX context;
+    SHA256_Init(&context);
+    SHA256_Update(&context, datum_name_.data(), datum_name_.size());
```

Comment:
ObNonCryptographerWarning: I have no idea whether this scheme makes cryptographic sense (e.g. are there preimage attacks that we'd need to worry about?)&hellip;

Reply:
We just need a deterministic digest, so this seems fine?